lib/src/components/NxTreeView/NxTreeView.tsx

```diff
@@ -4,92 +4,16 @@
-        treeViewClasses = classnames('nx-tree-view', className, {
-          'nx-tree-view--expanded': isExpanded,
-          'nx-tree-view--collapsed': !isExpanded,
-          'nx-tree-view--disabled': disabled,
-          'nx-tree-view--empty': isEmpty
-        }),
-        treeViewId = useMemo(() => id || getUniqueId('nx-tree-view'), [id]),
-        trigger = (
-          <button type="button"
-                  className="nx-tree-view__trigger"
-                  onClick={onToggleCollapse || undefined}
-                  aria-controls={treeViewId}
-                  aria-expanded={isExpanded}
-                  disabled={disabled || isEmpty || undefined}>
-            <NxFontAwesomeIcon className="nx-tree-view__twisty" icon={faCaretRight} />
-            <span className="nx-tree-view__text">
-              {ensureElement(triggerContent)}
-            </span>
-          </button>
-        ),
-        triggerTooltipProps = typeof triggerTooltip === 'string' ? { title: triggerTooltip } : triggerTooltip;
-
-    return (
-      <div className={treeViewClasses} id={treeViewId} role="list">
-        {
-          triggerTooltipProps ? (
-            // div necessary to avoid error message when tooltip is on disabled button
-            <NxTooltip { ...triggerTooltipProps } >
-              <div className="nx-tree-view__tooltip-wrapper">
-                {trigger}
-              </div>
-            </NxTooltip>
-          ) : trigger
-        }
-        <div className="nx-tree-view__children" role="group">
-          {children}
-        </div>
-      </div>
-    );
-  };
-
-/**
- * All individual treeview children should be wrapped in this component. When the child is an element,
- * this does not actually add another element to the DOM (as doing so would cause styling and screenreading
- * challenges) but instead adds the needed class and role to its child. If on the other hand the child is text,
- * this wraps it in a div
- */
-const NxTreeViewChild = forwardRef<Element, NxTreeViewChildProps>(
-    function NxTreeViewChildImpl({ children, className, ...otherProps }: NxTreeViewChildProps, ref) {
-      if (typeof children === 'string' || typeof children === 'number') {
-        return (
-          <NxTreeViewChild className={className} ref={ref} { ...otherProps }>
-            <div>{children}</div>
-          </NxTreeViewChild>
-        );
-      }
-      else {
-        const classes = classnames('nx-tree-view__child', children.props.className, className);
-
-        return React.cloneElement(children, { className: classes, ref, role: 'listitem', ...otherProps });
-      }
-    }
-);
-
-NxTreeViewChild.propTypes = childPropTypes;
-NxTreeView.propTypes = propTypes;
-
-export default NxTreeView;
-export { NxTreeViewChild };
+import NxCollapsibleItems, {
+  Props,
+  NxCollapsibleItemsChildProps,
+  NxCollapsibleItemsChild
+} from '../NxCollapsibleItems/NxCollapsibleItems';
+
+export default NxCollapsibleItems;
+
+export {
```

Comment:
We usually implement deprecated aliases right in `index.ts`

Reply:
> We usually implement deprecated aliases right in `index.ts`

Fixed:
f105440bd646d09311ba146eb3f3556b031e0384
f105440bd646d09311ba146eb3f3556b031e0384